contentreview-impl/impl/src/java/org/sakaiproject/contentreview/impl/turnitin/TurnitinReviewServiceImpl.java

```diff
@@ -3425,14 +3344,49 @@ private String switchLTIError(int result, String method){
 				return method + ": generic LTI error";
 		}
 	}
-	
-	private void processError(ContentReviewItem item, Long status, String error){//TODO resto de llamadas
-		item.setStatus(status);
-		if(error != null){
+
+	private void processError( ContentReviewItem item, Long status, String error, Integer errorCode )
+	{
+		if( status == null )
+		{
+			IllegalArgumentException ex = new IllegalArgumentException( "Status is null; you must supply a valid status to update when calling processError()" );
+			log.warn( ex );
```

Comment:
Doesn't this result in double logging of the error as normally if the caller gets an Exception it should log it somewhere?